README.md

```diff
@@ -1,87 +1,90 @@
-# Aliyun OSSUTIL 
-[![GitHub version](https://badge.fury.io/gh/aliyun%2Fossutil.svg)](https://badge.fury.io/gh/aliyun%2Fossutil)
+# Alibaba Cloud OSSUTIL
+ 
+[![GitHub Version](https://badge.fury.io/gh/aliyun%2Fossutil.svg)](https://badge.fury.io/gh/aliyun%2Fossutil)
 [![Build Status](https://travis-ci.org/aliyun/ossutil.svg?branch=master)](https://travis-ci.org/aliyun/ossutil)
 [![Coverage Status](https://coveralls.io/repos/github/aliyun/ossutil/badge.svg?branch=master)](https://coveralls.io/github/aliyun/ossutil?branch=master)
-## 关于
-> - 此工具采用go语言，基于OSS[阿里云对象存储服务](http://www.aliyun.com/product/oss/)官方GO SDK 1.4.0 构建。
-> - 阿里云对象存储（Object Storage Service，简称OSS），是阿里云对外提供的海量，安全，低成本，高可靠的云存储服务。
-> - OSS适合存放任意文件类型，适合各种网站、开发企业及开发者使用。
-> - 该工具旨在为用户提供一个方便的，以命令行方式管理OSS数据的途径。
-> - 当前版本提供了列举和删除Multipart Uploads功能。
-> - 当前版本未提供Bucket管理功能功能，相关功能会在后续版本中开发。
-
-## 版本
-> - 当前版本：1.2.0
-
-## 运行环境
-> - linux, windows 
-
-## 依赖的库 
-> - goopt (github.com/droundy/goopt) 
-> - configparser (github.com/alyu/configparser)
-> - leveldb (github.com/syndtr/goleveldb/leveldb)
-> - oss (github.com/aliyun/aliyun-oss-go-sdk/oss)
-> - gopkg.in/check.v1 (gopkg.in/check.v1)
-
-## 快速使用
-#### 获取命令列表
+
+### [README of Chinese](https://github.com/aliyun/ossutil/blob/master/README-CN.md)
+
+## About
+- This tool is developed with Go and built on the official GO SDK 1.6.0 of OSS [Alibaba Cloud Object Storage Service](http://www.aliyun.com/product/oss/).
+- Alibaba Cloud Object Storage Service (OSS) is a cloud storage service provided by Alibaba Cloud, featuring a massive capacity, security, a low cost, and high reliability.
+- The OSS can store any type of files and therefore applies to various websites, development enterprises and developers.
```

Comment:
Following will be better.

OSS can store any type of files. It applies to various ...

Reply:
fixed